config/config.yaml

```diff
@@ -53,16 +57,19 @@ cacheConfig:
   slotsPerInstance: ${KALDB_CACHE_SLOTS_PER_INSTANCE:-10}
   dataDirectory: ${KALDB_CACHE_DATA_DIR:-/tmp}
   maxParallelCacheSlotDownloads: ${KALDB_CACHE_MAX_PARALLEL_CACHE_SLOT_DOWNLOADS:-3}
+  defaultQueryTimeoutMs: ${DEFAULT_QUERY_TIMEOUT_MS:-2500}
```

Comment:
Probably want to prefix these with the component like the other envs. This way if you want to run KalDB in a single JVM you can still make use of different timeout values per service.